src/Microsoft.AspNet.Server.Kestrel/Http/FrameResponseStream.cs

```diff
@@ -23,13 +23,7 @@ public FrameResponseStream(FrameContext context)
 
         public override bool CanWrite => true;
 
-        public override long Length
-        {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
+        public override long Length => Position;
```

Comment:
Maybe we just add a new property to FrameResponseStream. `BytesWritten` perhaps?

@lodejard where were you planning on logging the this?